python/tests/data/svg/mut_tree.svg

```diff
@@ -0,0 +1 @@
+<svg baseProfile="full" height="200" id="XYZ" version="1.1" width="200" xmlns="http://www.w3.org/2000/svg" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><style type="text/css"><![CDATA[.axis {font-weight: bold}.tree, .axis {font-size: 14px; text-anchor:middle}.edge {stroke: black; fill: none}.node > .sym {fill: black; stroke: none}.tree .lab {dominant-baseline: middle}.tree .lab.lft {text-anchor: end}.tree .lab.rgt {text-anchor: start}.mut {fill: red; font-style: italic}]]></style><style type="text/css"><![CDATA[.edge {stroke: blue}]]></style></defs><g class="tree t0"><g class="m0 m1 node n9 p0 root s0" transform="translate(100.0 50.0)"><g class="a9 m2 node n4 p0 s0" transform="translate(-36.0 119.677)"><g class="a4 leaf node n0 p0 sample" transform="translate(-18.0 0.32302)"><path class="edge" d="M 0 0 V -0.32302 H 18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">0</text></g><g class="a4 leaf node n1 p0 sample" transform="translate(18.0 0.32302)"><path class="edge" d="M 0 0 V -0.32302 H -18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">1</text></g><path class="edge" d="M 0 0 V -119.677 H 36.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab lft" transform="translate(-3 -6)">4</text><g class="mut m2 s0" transform="translate(0 -59.8385)"><rect class="sym" height="6" transform="translate(-3 -3)" width="6" x="0" y="0" /><text class="lab lft" transform="translate(-5 0)">2</text></g></g><g class="a9 node n5 p0" transform="translate(36.0 118.538)"><g class="a5 leaf node n2 p0 sample" transform="translate(-18.0 1.46223)"><path class="edge" d="M 0 0 V -1.46223 H 18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">2</text></g><g class="a5 leaf node n3 p0 sample" transform="translate(18.0 1.46223)"><path class="edge" d="M 0 0 V -1.46223 H -18.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab" transform="translate(0 12)">3</text></g><path class="edge" d="M 0 0 V -118.538 H -36.0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab rgt" transform="translate(3 -6)">5</text></g><path class="edge" d="M 0 0 V -20.0 H 0" /><circle class="sym" cx="0" cy="0" r="3" /><text class="lab rgt" transform="translate(3 -6)">9</text><g class="mut m1 s0" transform="translate(0 -6.66667)"><rect class="sym" height="6" transform="translate(-3 -3)" width="6" x="0" y="0" /><text class="lab rgt" transform="translate(5 0)">1</text></g><g class="mut m0 s0" transform="translate(0 -13.3333)"><rect class="sym" height="6" transform="translate(-3 -3)" width="6" x="0" y="0" /><text class="lab rgt" transform="translate(5 0)">0</text></g></g></g></svg>
```

Comment:
It was nice having these examples files in a more readable format - any chance we can output then in the same way?

Reply:
That's a good point. I'll work out how to do that.

Reply:
I think there's a "debug" option in svgwrite which pretty-prints the XML. You've probably turned this off somewhere.

Reply:
Looks like `debug=True` in the code, which is [not recommended in production code](https://svgwrite.readthedocs.io/en/latest/overview.html?highlight=debug#overview). So should we add it as an option to `draw_svg()` and default to "False"? (this doesn't explain why it's not outputting like that, though)

Reply:
You could run them through `tidy`?

Reply:
> Looks like debug=True in the code, which is not recommended in production code. So should we add it as an option to draw_svg() and default to "False"? (this doesn't explain why it's not outputting like that, though)

I don't think the speed is a real issue, so I'd leave ``debug=True`` alone for now. But you're right it doesn't explain why the files have changed. Possibily because svgwrite's behaviour has changed. Ben's right, let's run it through an external XML formatter.

Reply:
Any preferences for XML formatters? Are we worried about an extra dev requirement? I guess not.

Reply:
It's only a requirement for someone who is editing the example SGVs. Would be nice to add a comment to the SVG saying how it was formatted to avoid future hand-editing. No preference as long as it is widely avaliable.

Reply:
Looks like there is a builtin version: `xml.dom.minidom` + `xml.toprettyxml()`